webapp/src/shared/components/Game/hooks/useOnlineGame.tsx

```diff
@@ -26,11 +26,15 @@ export function useOnlineGame(gameId: string) {
   const [gameJoined, setGameJoined] = React.useState(false);
 
   const sendGameCommand = React.useCallback(
-    (command: GameCommand) => {
+    (
+      command: GameCommand,
+      payload?: { dragCurrPos: number; dragPrevPos: number },
```

Comment:
I'm going to say what I'm always saying: why don't we type this explicitly? 
```
class DragPayload {
  @IsNumber()
  dragCurrPos!: number;

  @IsNumber()
  dragPrevPos!: number;
}
```
seems to be just two numbers...
Although I'm not sure about the GameInputPayload...

You tell me.

Reply:
> I'm going to say what I'm always saying: why don't we type this explicitly?

Done. Please let me know what you think.

> seems to be just two numbers...

I'm not sure what you mean.

Reply:
Just two numbers means that we don't have the problem we had yesterday with the User type.

Reply:
About the most recent changes: you can also put the class validator in the pong engine as well, and validate your object.

Reply:
Thanks :+1:
I prefer not to add the class validator library to the pong engine because it doesn't use it, and the validation happens on the server side.